data/clean_shakespear/clean_shakespear.txt

```diff
@@ -0,0 +1 @@
d the common? Cominius, no. Have I had children's voices? Tribunes, give way; he shall to the market-place. The people are incensed against him. Stop, Or all will fall in broil. Are these your herd? Must these have voices, that can yield them now And straight disclaim their tongues? What are your offices? You being their mouths, why rule you not their teeth? Have you not set them on? Be calm, be calm. It is a purposed thing, and grows by plot, To curb the will of the nobility: Suffer't, and live with such as cannot rule Nor ever will be ruled. Call't not a plot: The people cry you mock'd them, and of late, When corn was given them gratis, you repined; Scandal'd the suppliants for the people, call'd them Time-pleasers, flatterers, foes to nobleness. Why, this was known before. Not to them all. Have you inform'd them sithence? How! I inform them! You are like to do such business. Not unlike, Each way, to better yours. Why then should I be consul? By yond clouds, Let me deserve so ill as you, and make me Your fellow tribune. You show too much of that For which the people stir: if you will pass To where you are bound, you must inquire your way, Which you are out of, with a gentler spirit, Or never be so noble as a consul, Nor yoke with him for tribune. Let's be calm. The people are abused; set on. This paltering Becomes not Rome, nor has Coriolanus Deserved this so dishonour'd rub, laid falsely I' the plain way of his merit. Tell me of corn! This was my speech, and I will speak't again-- Not now, not now. Not in this heat, sir, now. Now, as I live, I will. My nobler friends, I crave their pardons: For the mutable, rank-scented many, let them Regard me as I do not flatter, and Therein behold themselves: I say again, In soothing them, we nourish 'gainst our senate The cockle of rebellion, insolence, sedition, Which we ourselves have plough'd for, sow'd, and scatter'd, By mingling them with us, the honour'd number, Who lack not virtue, no, nor power, but that Which they have given to beggars. Well, no more. No more words, we beseech you. How! no more! As for my country I have shed my blood, Not fearing outward force, so shall my lungs Coin words till their decay against those measles, Which we disdain should tatter us, yet sought The very way to catch them. You speak o' the people, As if you were a god to punish, not A man of their infirmity. 'Twere well We let the people know't. What, what? his choler? Choler! Were I as patient as the midnight sleep, By Jove, 'twould be my mind! It is a mind That shall remain a poison where it is, Not poison any further. Shall remain! Hear you this Triton of the minnows? mark you His absolute 'shall'? 'Twas from the canon. 'Shall'! O good but most unwise patricians! why, You grave but reckless senators, have you thus Given Hydra here to choose an officer, That with his peremptory 'shall,' being but The horn and noise o' the monster's, wants not spirit To say he'll turn your current in a ditch, And make your channel his
```

Comment:
Is there a value in keeping the full cleaned .txt file in the repo if we have the .zip?

Reply:
True, I'll just leave the zip